core/src/test/resources/org/elasticsearch/index/analysis/test1.json

```diff
@@ -35,10 +35,6 @@
                 "custom6":{
                     "tokenizer":"standard",
                     "position_increment_gap": 256
-                },
```

Comment:
This analyzer with czech stem token filter wasn't actually being used. Besides that the commented out test was redundant and the dependency on this token filter is unneeded in this test case.